draft-ietf-quic-qpack.md

```diff
@@ -742,19 +747,15 @@ the existing entry is represented as an integer with a 5-bit prefix.
 {:#fig-index-with-duplication title="Duplicate"}
 
 The existing entry is re-inserted into the dynamic table without resending
-either the name or the value. This is useful to mitigate the eviction of older
-entries which are frequently referenced, both to avoid the need to resend the
-header and to avoid the entry in the table blocking the ability to insert new
-headers.
+either the name or the value. This is useful to avoid adding a reference to an
+older entry, which might block inserting new entries.
 
 
 ## Decoder Instructions {#decoder-instructions}
 
-Decoder instructions provide information used to ensure consistency of the
```

Comment:
I propose to change this paragraph to match the first paragraph of Encoder Instructions section.